simulator/src/main/java/peersim/transport/PairwiseFixedLatencyTransport.java

```diff
@@ -0,0 +1,65 @@
+package peersim.transport;
+
+import peersim.config.*;
+import peersim.core.*;
+import peersim.edsim.*;
+
+public final class PairwiseFixedLatencyTransport extends UniformRandomTransport {
+
+  /** Latency between nodes (symmetric). */
+  private static int[][] array;
```

Comment:
i think this is not used, so can be removed

Reply:
yes, removed it.